src/views/dashboard/Pam/RiskSummary.vue

```diff
@@ -204,8 +204,8 @@ export default {
         params: {
           total_long_time_no_login_accounts: 1,
           total_new_found_accounts: 1,
-          total_group_changed_accounts: 1,
-          total_sudo_changed_accounts: 1,
+          total_groups_changed_accounts: 1,
+          total_sudoers_changed_accounts: 1,
           total_authorized_keys_changed_accounts: 1,
           total_password_expired_accounts: 1,
           total_long_time_password_accounts: 1,
```

Comment:
There are no obvious issues. The code is clean and does not contain any errors related to the requirements of knowledge cutoff (2021-09-01) and current date (2025-03-17) provided. No further adjustments required for optimal performance.